app/views/contacts/index.html.erb

```diff
@@ -3,12 +3,20 @@
 <th>First name</th>
 <th>Email</th>
 <th># of appointments</th>
+<th>Most Recent Appointment</th>
 
 <% @contacts.each do |contact| %>
   <tr>
   <td><%= contact.first_name %></td>
   <td><%= contact.email %></td>
+  <!-- td><%#= contact.most_recent_appointment %></td -->
+
   <td><%= contact.appointments.size %></td>
+  <% if contact.appointments.size > 0 %>
```

Comment:
let's get this logic out of the controller -- seems like it should be using a method on `contact` to find the most recent appointment rather than having the ruby in here